code/modules/projectiles/guns/projectile/automatic/luty.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/gun/projectile/automatic/luty
+    name = "handmade SMG .35 Auto \"Luty\""
+    desc = "A dead simple open-bolt automatic firearm, easily made and easily concealed.\
+            A gun that has gone by many names, from the Grease gun to the Carlo to the Swedish K. \
+            Some designs are too good to change."
+    icon = 'icons/obj/guns/projectile/luty.dmi'
+    icon_state = "luty"
+    item_state = "luty"
+
+    w_class = ITEM_SIZE_NORMAL
+    can_dual = TRUE
+    caliber = CAL_PISTOL
+    slot_flags = SLOT_BELT|SLOT_HOLSTER
+    ammo_type = /obj/item/ammo_casing/pistol
+    load_method = MAGAZINE
+    mag_well = MAG_WELL_PISTOL|MAG_WELL_H_PISTOL|MAG_WELL_SMG
+    magazine_type = /obj/item/ammo_magazine/smg
+
+    init_firemodes = list(
+        FULL_AUTO_400,
+        SEMI_AUTO_NODELAY,
+        )
+
+    can_dual = 1
+    damage_multiplier = 0.7
+    penetration_multiplier = 0.9
+    recoil_buildup = 1
+    one_hand_penalty = 5 //SMG level.
+    spawn_blacklisted = TRUE
+
+    origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2)
+    matter = list(MATERIAL_STEEL = 15, MATERIAL_WOOD = 10)
+
```

Comment:
You forgot to override gun_parts , set it to gun_parts = list(MATERIAL_STEEL = 15) or something so people can't make the gun , then wrench it for gun components.

Reply:
there won't be any gun part items that would be gained, since it gets gun_parts from its parent

Reply:
![image](https://user-images.githubusercontent.com/61350382/133928965-a53c43f6-cf81-400a-a4c8-1ea20964b87b.png)

Override it.

Reply:
done